registry/README.md

```diff
@@ -4,37 +4,37 @@
 
 ### Build Package
 
-Create `azpkg` and upload to a ßstorage account.
-
-```powershell
-PackageGenerator\AzureGalleryPackager.exe -m <path-to-manifest.json> -o <output-dir>
-```
+Use the `make release` command to create `azpkg` file for Docker Container Registry Marketplace Item, and find it in the "_out" folder. Upload the file to a blob container in a storage account, and make sure to change the access policy of the blob to enable anonymous access.
```

Comment:
nit

- it should be `make release` TARGET, not command.
- "to create an Azure Gallery Package (.azpkg) for the Docker Container Registry solution template".
- "the package will be placed in the `_out` directory"
- "update the`package` to ..."